lib/themes/dosomething/paraneue_dosomething/js/components/InstructionSlide.js

```diff
@@ -27,8 +27,8 @@ class InstructionSlide extends React.Component {
 
             <div className="container__row">
               <div className="container__block -narrow -centered">
-                <div className="beta-card">
-                  <video className="beta-card__video" poster="https://static.dosomething.org/onboarding/poster.png" src="https://static.dosomething.org/onboarding/CouldBeYou_720p.mp4" loop autoPlay controls width="100%"></video>
+                <div className="media-video">
```

Comment:
We didn't really need the `beta-card` here _just_ for the border-radius. Chatted with @lkpttn and looks like we want to use border-radius moving forward on a lot of our "media" so maybe we add it to the `media-*` patterns. Left a `@TODO` in the comments.